pymodbus/server/simulator/http_server.py

```diff
@@ -301,14 +302,18 @@ async def handle_html(self, request):
     async def handle_json(self, request):
         """Handle api registers."""
         page_type = request.path.split("/")[-1]
-        params = await request.post()
-        json_dict = self.generator_html[page_type][0].copy()
-        result = self.generator_json[page_type][1](params, json_dict)
-        return web.Response(text=f"json build: {page_type} - {params} - {result}")
+        params = await request.json()
+        json_dict = copy.deepcopy(self.generator_json[page_type][0])
```

Comment:
what happened in page_type is unknown ??

the name "page_type" does not relate well to a RestApi request, did you mean "request"

Reply:
> what happened in page_type is unknown ??

The aiohttp handles the unregistered endpoint. Similar to what happens when using html.

> the name "page_type" does not relate well to a RestApi request, did you mean "request"

Did not pay any mind to it. I'll change it.